examples/basic.rs

```diff
@@ -3,7 +3,7 @@ extern crate hashicorp_vault as vault;
 fn main() {
     let host = "http://localhost:8200";
     let token = "test12345";
-    let client = vault::Client::new(host, token).unwrap();
+    let client = vault::Client::new(host, token, &[""]).unwrap();
```

Comment:
Yeah, I think that's a good idea as well. There's also https://github.com/colin-kiegel/rust-derive-builder which could help (I haven't used it though).